src/components/listitems/ListItemSwitch.tsx

```diff
@@ -20,6 +20,7 @@ type PartialProps = {
   action?: SwitchAction;
   isLoading?: boolean;
   badge?: Badge;
+  testID?: string;
```

Comment:
Just a "cosmetic" suggestion: what about using the `withTestID` util function?

Reply:
Good one, applied